README.rst

```diff
@@ -31,6 +31,10 @@ point and all grid points. With the 'functional' drift capability, the user may
 of the spatial coordinates that define the drift(s). The package includes a module that contains functions
 that should be useful in working with ASCII grid files (`*.asc`).
 
+PyKrige also works smoothly together with the Covariance Model provided by
+`GSTools <https://github.com/GeoStat-Framework/GSTools>`_ v1.1 and higher.
+Have a look at the section: `GSTools covariance models`_.
```

Comment:
I would remove this last sentence, as a link to the documentation of another package before the link to pykrige documentation is confusing. User who need this functionality would be able to find the instructions from other links in this PR.